instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -284,16 +286,22 @@ def get_or_create_headers():
                 ).__qualname__
 
             if duration_histogram_old is not None:
-                duration_attrs_old = _filter_duration_attrs(
-                    metric_labels, _OpenTelemetryStabilityMode.DEFAULT
+                duration_attrs_old = _filter_semconv_duration_attrs(
+                    metric_labels,
+                    _client_duration_attrs_old,
+                    _client_duration_attrs_new,
+                    _HTTPStabilityMode.DEFAULT
```

Comment:
Seems hardocded, shouldn't this value depend on that environment variable value?

Reply:
No, we have two histograms representing the old and new semantic conventions.